lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -884,24 +897,69 @@ def _post_start_init(self, event: EventBase):
 
         self.node_lock.release()
 
-        self._upgrade.unit_state = upgrade.UnitState.HEALTHY
-        self._reconcile_upgrade()
+        # TODO upgrade: catch http errors
+        self.opensearch.request(
+            "PUT",
+            "/_all/_settings",
+            # Reset to default value
+            payload={"settings": {"index.unassigned.node_left.delayed_timeout": None}},
```

Comment:
I think we need to specify the level of `settings`. In the OS docs, it recommends a `persistent` setting:
```
PUT "/_cluster/settings?pretty"
{
    "persistent": {
        "cluster.routing.allocation.enable": "primaries"
    }
}
```